cfme/utils/ext_auth.py

```diff
@@ -1,127 +0,0 @@
-# -*- coding: utf-8 -*-
-import fauxfactory
-from time import sleep
-
-from cfme.utils.browser import ensure_browser_open
-from cfme.utils.conf import credentials
-from cfme.utils.ssh import SSHClient
-from cfme.utils.appliance import get_or_create_current_appliance
-from cfme.utils.path import conf_path
-
-# TODO move these into appliance
-
```

Comment:
Everything moved to appliance methods, auth provider classes, or new auth fixture.